tests/test-cases/Group23-VIC-Machine-Service/23-06-VCH-Certificate.robot

```diff
@@ -0,0 +1,103 @@
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.esxcli network firewall set -e false
+    # Attempt to cleanup old/canceled tests
+    Run Keyword If  ${cleanup}  Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword If  ${cleanup}  Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    Run Keyword If  ${cleanup}  Run Keyword And Ignore Error  Cleanup Dangling Networks On Test Server
+    Run Keyword If  ${cleanup}  Run Keyword And Ignore Error  Cleanup Dangling vSwitches On Test Server
+    Run Keyword If  ${cleanup}  Run Keyword And Ignore Error  Cleanup Dangling Containers On Test Server
+    Run Keyword If  ${cleanup}  Run Keyword And Ignore Error  Cleanup Dangling Resource Pools On Test Server
+    Set Suite Variable  ${vicmachinetls}  --no-tls
+    Log To Console  \nInstalling VCH to test server with tls disabled...
+    ${output}=  Run VIC Machine Command  ${vic-machine}  ${appliance-iso}  ${bootstrap-iso}  ${certs}  ${vol}  ${debug}  ${additional-args}
+    Log  ${output}
+    Should Contain  ${output}  Installer completed successfully
+
+    Get Docker Params  ${output}  ${certs}
+    Log To Console  Installer completed successfully: %{VCH-NAME}...
+
+    [Return]  ${output}
+
+Start VIC Machine Server
+    Start Process  ./bin/vic-machine-server --port 31337  shell=True  cwd=/go/src/github.com/vmware/vic
+
+Curl No Datacenter
+    [Arguments]  ${vch-id}  ${auth}
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -s -w "\%{http_code}\n" -X GET "http://127.0.0.1:31337/container/target/%{TEST_URL}/vch/${vch-id}/certificate?thumbprint=%{TEST_THUMBPRINT}" -H "authorization: Basic ${auth}"
+    [Return]  ${rc}  ${output}
+
+Curl Datacenter
+    [Arguments]  ${vch-id}  ${auth}
+    ${orig}=  Get Environment Variable  TEST_DATACENTER
+    ${dc}=  Run Keyword If  '%{TEST_DATACENTER}' == '${SPACE}'  Get Datacenter Name
+    Run Keyword If  '%{TEST_DATACENTER}' == '${SPACE}'  Set Environment Variable  TEST_DATACENTER  ${dc}
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -s -w "\%{http_code}\n" -X GET "http://127.0.0.1:31337/container/target/%{TEST_URL}/datacenter/%{TEST_DATACENTER}/vch/${vch-id}/certificate?thumbprint=%{TEST_THUMBPRINT}" -H "authorization: Basic ${auth}"
+    Set Environment Variable  TEST_DATACENTER  ${orig}
+    [Return]  ${rc}  ${output}
+
+*** Test Cases ***
+Get VCH Certificate
+    Install VIC Appliance To Test Server
+    ${id}=  Get VCH ID  %{VCH-NAME}
+    ${auth}=  Evaluate  base64.b64encode("%{TEST_USERNAME}:%{TEST_PASSWORD}")  modules=base64
+    ${rc}  ${output}=  Curl No Datacenter  ${id}  ${auth}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  BEGIN CERTIFICATE
+    Should Contain  ${output}  END CERTIFICATE
+    ${rc}  ${outputDC}=  Curl Datacenter  ${id}  ${auth}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  BEGIN CERTIFICATE
+    Should Contain  ${output}  END CERTIFICATE
```

Comment:
I think lines 73 and 74 were intended to reference `outputDC`, not `output`.

That said, we check that `output` and `outputDC` are equal on the next line, so maybe we don't need to make the same assertions about both?

Reply:
Good catch.. I cleaned up both tests along these lines.